apis/components/v1alpha1/datasciencepipelines_types.go

```diff
@@ -51,6 +52,7 @@ type DataSciencePipelinesSpec struct {
 
 type DataSciencePipelinesCommonSpec struct {
 	common.DevFlagsSpec `json:",inline"`
+	PreloadedPipelines  datasciencepipelines.PreloadedPipelinesSpec `json:"preloadedPipelines,omitempty"`
```

Comment:
@HumairAK do you think that we should change the term "preloaded" here since we will be supporting importing the pipeline on existing pipeline servers? Some ideas are `builtinPipelines`, `managedPipelines`, or `platformPipelines`.

Reply:
agreed `managedPipelines` sounds the most appropriate to me

Reply:
Done in c191dbb2497de6a0097e3622f2e086361bf5d251